coa/pkg/logger/contexts/activitylog-context.go

```diff
@@ -0,0 +1,489 @@
+package contexts
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/valyala/fasthttp"
+)
+
+const (
+	ACTIVITY_HTTP_HEADER_PREFIX string = "X-Activity-"
+	Activity_ResourceCloudId    string = "resourceId"
+	Activity_OperationName      string = "operationName"
+	Activity_Location           string = "location"
+	Activity_Category           string = "category"
+	Activity_CorrelationId      string = "correlationId"
+	Activity_Properties         string = "properties"
+
+	Activity_Props_CallerId      string = "caller-id"
```

Comment:
Do we need both Activity_Props_CallerId and OTEL_Activity_Props_CallerId

Reply:
The intention is we can allow different key in logrus and when sending to otel-collector. So far, these are the same.